libs/sdk-ui/src/base/results/internal/dataAccessImpl.ts

```diff
@@ -433,7 +433,7 @@ export class DataAccessImpl {
             measureTitle: (): string => {
                 return measureName(measureDescriptor);
             },
-            scopeTitles: (): string[] => {
+            scopeTitles: (): (string | null)[] => {
```

Comment:
Please be consistent: pick either `Array<string | null>` or `(string | null)[]` and use only one.

Reply:
fixed